libsolidity/inlineasm/AsmCodeGen.cpp

```diff
@@ -20,14 +20,15 @@
  * Code-generating part of inline assembly.
  */
 
-#include <libsolidity/inlineasm/AsmCodeGen.h>
```

Comment:
It seemed from the CodingStandards.txt that the order was as this.  I've missed the `point b.`:
> b. The only exception to the above rule is the top of a .cpp file where its corresponding header should be located.